constraints/miden-vm/hash.air

```diff
@@ -144,19 +160,24 @@ ev node_index(main: [s[3], i]):
     let f_an = f_mp + f_mv + f_mu + f_mpa + f_mva + f_mua
 
     # Enforce that b is binary only when a new node is absorbed into the hasher state.
+    # Constraint degree: 6
     enf f_an * (b^2 - b) = 0
 
     # Enforce that when a computation is finished i = 0.
+    # Constraint degree: 4
     enf f_out * i = 0
```

Comment:
Similar comment to the one above: I think the degree of this constraint should be 4, not 5.